src/_styles/style.scss

```diff
@@ -11,7 +11,18 @@
 /* Cards */
 @import 'cards/ArticleCard';
 
+/* Components */
+@import 'components/FiveStars';
+@import 'components/Toast';
+
 /* Pages */
 @import 'pages/Login';
 @import 'pages/Profile';
+<<<<<<< HEAD
```

Comment:
Conflicts were not resolved properly, modify this file

Reply:
@mcaleb808 Thanks for the feedback